settings.gradle.kts

```diff
@@ -125,6 +125,7 @@ include(":e2e-tests:runtimes:sts")
 include(":e2e-tests:bom-tests")
 include(":e2e-tests:fixtures")
 include(":e2e-tests:admin-api-tests")
+include(":e2e-tests:dcp-tests")
```

Comment:
minor nit: i'd call this `:dcp-issuance-tests`, because we probably should move the current `PesentationApiEndToEndTest` into a similar `:dcp-presentation-tests` module, wdyt?